First_section.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="First_section.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <header class="toolbar">
+        <h1 id="Logo">My Logo</h1>
+        <img id="menu" src="union.png" />
+      </header>
+
+      <main class="content">
```

Comment:
- [ ] The whole of this should be in a section tag. The section represents a generic standalone section of the document.

Reply:
Please, check the site you are using to review the html. on my end, the background shows and the icon is visible. i have tested even in the chrome. everything is visible. thank you for your kind feedback. 

<img width="499" alt="Screenshot 2022-12-01 at 18 00 24" src="https://user-images.githubusercontent.com/43499747/205100677-959aadb2-228f-4119-bc05-14a10c3eb348.png">